versionedRouter.js

```diff
@@ -587,12 +609,12 @@ router.post("/batch", ctx => {
   batchHandler(ctx);
 });
 
-router.get("/heapdump", ctx => {
-  heapdump.writeSnapshot((err, filename) => {
-    logger.debug("Heap dump written to", filename);
-  });
-  ctx.body = "OK";
-});
+// router.get("/heapdump", ctx => {
+//   heapdump.writeSnapshot((err, filename) => {
```

Comment:
This shouldn't be commented right ?!

Reply:
We are not using this and every time we debug we have to commented this hence I have commented this. We can remove this actually